lldb/include/lldb/Core/Telemetry.h

```diff
@@ -28,6 +28,17 @@
 namespace lldb_private {
 namespace telemetry {
 
+struct LLDBConfig : public ::llvm::telemetry::Config {
+  // If true, we will collect telemetry from LLDB's clients (eg., lldb-dap) via
+  // the SB interface. Must also be enabled by the vendor while creating the
+  // manager.
+  const bool m_enable_client_telemetry;
```

Comment:
Sorry to chime in but I'm just new to this project and I'm wonder why you effectively need a flag.  If the client needs to send manually a telemetry event, isn't this redudant?